### DISCUSSION TITLE
Please allow disabling of auto-launching web GUI.

### DISCUSSION THREAD

**Discussion by FoxyDrew** [rejetto/hfs#673] [Ideas]
I use the latest Windows version from the releases page, rather than fully installing Node.js and doing a manual install.

I'm completely content with this and would prefer it, it's just really annoying that every time I launch the "hfs.exe" it'll automatically open the WebUI in my browser. 

I've set everything up how I've wanted already, and don't need the webGUI to launch every time I launch the program. A config option for this would be amazing and greatly appreciated.

**Comment by rejetto** [comment]
it's already in options
<img width="270" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/01b91fb5-b7f3-4a76-9af0-64a09411e666">
